packages/cli/oclif.manifest.json

```diff
@@ -1567,7 +1567,7 @@
           "description": "Resource URL for checkout UI extension. Format: \"/cart/{productVariantID}:{productQuantity}\"",
           "env": "SHOPIFY_FLAG_CHECKOUT_CART_URL",
           "hasDynamicHelp": false,
-          "hidden": false,
+          "hidden": true,
```

Comment:
I could have removed these flags from this file entirely too instead of setting `hidden` to `true` but I figured no harm in keeping them if they ever become relevant to display in future. Let me know if folks who have more context than me think otherwise.

Reply:
I want to avoid the scenario where someone copies over this command from dev and expects it to do something, only for it to quietly do nothing- I think full removal is necessary